package-lock.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
and this file should not exist at all :) you must have run `npm install` from the root folder, which is incorrect.